packages/android-client/FishjamClient/build.gradle

```diff
@@ -0,0 +1,113 @@
+plugins {
+    id 'com.android.library'
+    id 'org.jetbrains.kotlin.android'
+    id "org.jetbrains.dokka"
+    id "com.google.protobuf" version "0.9.4"
+    id 'maven-publish'
+    id "kotlin-android"
+}
+
+ext {
+      compose_version = '1.2.0-alpha04'
+      compose_compiler_version = '1.1.0-rc02'
+      kotlin_version = '1.8.0'
+      java_version = JavaVersion.VERSION_17
+      dokka_version = '1.8.10'
+}
```

Comment:
this was copied from the upper-level gradle file, can we avoid this somehow?

also in the upper gradle file there is java_11, won't it break something?

Reply:
i copied it from code you gave me 😄 
I can check if it can be deleted.

Reply:
i unifided variables and removed not neccesary ones